examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.cpp

```diff
@@ -604,7 +601,7 @@ static void wfx_rsi_save_ap_info(void) // translation
     sl_wifi_ssid_t ssid_arg;
     memset(&ssid_arg, 0, sizeof(ssid_arg));
     ssid_arg.length = wfx_rsi.sec.ssid_length;
-    chip::Platform::CopyString((char *) &ssid_arg.value[0], ssid_arg.length, wfx_rsi.sec.ssid);
+    memcpy((char *) &ssid_arg.value[0], wfx_rsi.sec.ssid, ssid_arg.length);
```

Comment:
why are we replacing CopyString by memcpy? Copy string adds some verification/safe checks

Reply:
Updated the PR